lib/provider/ui/windows/control-panel.tsx

```diff
@@ -0,0 +1,69 @@
+import type { ReactNode } from "react";
+import type { SelectOption } from "react95/dist/Select/Select.types";
+import type { Theme } from "react95/dist/types";
+
+import { useAtom } from "jotai";
+import { useEffect, useState } from "react";
+import { GroupBox, Select } from "react95";
+import styled from "styled-components";
+
+import { themeAtom } from "~/lib/atom";
+import { Window } from "~/lib/window";
+
+type ThemeType = "original" | "blackAndWhite" | "modernDark";
+
+const themeOptions: SelectOption<ThemeType>[] = [
+    { label: "Default", value: "original" },
+    { label: "Black and white", value: "blackAndWhite" },
+    { label: "Modern Dark", value: "modernDark" }
+];
+
+const StyledWindow = styled(Window)`
+  overflow: visible;
+`;
+
+export function ControlPanel(): ReactNode {
+    const [theme, setTheme] = useAtom(themeAtom);
+    const [selectedTheme, setSelectedTheme] = useState<ThemeType>("original");
+    const [themesMap, setThemesMap] = useState<Record<ThemeType, Theme>>({} as Record<ThemeType, Theme>);
+
+    useEffect(() => {
+        const loadThemes = async (): Promise<void> => {
+            const themes = await import("react95/dist/themes");
+
+            setThemesMap({
+                original: themes.default.original,
+                blackAndWhite: themes.default.blackAndWhite,
+                modernDark: themes.default.modernDark,
+            });
```

Comment:
We should not hard code this, it limits the number of themes we'll have. Instead it should be `setThemes(await imoort("react95/dist/themes")`